scripts/_common.sh

```diff
@@ -121,29 +117,21 @@ install_jellyfin_discovery_ports() {
 
 		# Open port $discovery_service_port for service auto-discovery
 		if ynh_port_available --port=$discovery_service_port; then
-			ynh_exec_warn_less yunohost firewall allow UDP $discovery_service_port
+			ynh_hide_warnings yunohost firewall allow UDP $discovery_service_port
 		else
 			discovery_service=0
-			ynh_print_warn --message="Port $discovery_service_port (for service auto-discovery) is not available. Continuing nonetheless."
+			ynh_print_warn "Port $discovery_service_port (for service auto-discovery) is not available. Continuing nonetheless."
 		fi
 
 		# Open port $discovery_client_port for client auto-discovery
 		if ynh_port_available --port=$discovery_client_port; then
-			ynh_exec_warn_less yunohost firewall allow UDP $discovery_client_port
+			ynh_hide_warnings yunohost firewall allow UDP $discovery_client_port
 		else
 			discovery_client=0
-			ynh_print_warn --message="Port $discovery_client_port (for client auto-discovery) is not available. Continuing nonetheless."
+			ynh_print_warn "Port $discovery_client_port (for client auto-discovery) is not available. Continuing nonetheless."
 		fi
 	fi
 
-	ynh_app_setting_set --app="$app" --key=discovery_service --value="$discovery_service"
-	ynh_app_setting_set --app="$app" --key=discovery_client --value="$discovery_client"
+	ynh_app_setting_set --key=discovery_service --value="$discovery_service"
+	ynh_app_setting_set --key=discovery_client --value="$discovery_client"
 }
```

Comment:
I’m not sure what to do with those helpers.
ynh_port_available has been remove from helpers2.1

Should I just enable the ports in the manifest? But how to make it conditional like now?